Add doc for remote debugging in vscode

isabellachen opened the pull request:
Fixes #

Related to #13180 

docker/Readme.md

Add documentation for remote debugging in VSCode.

isabellachen committed:
Add doc for remote debugging in vscode

isabellachen committed:
[not verified] Add doc for remote debugging in vscode

isabellachen committed:
Updated readme

isabellachen committed:
[not verified] Updated readme

brbrr commented on docker/README.md:
```diff
@@ -458,3 +458,60 @@ Below are instructions for starting a debug session in PhpStorm that will listen
+
+You'll need:
+
+- PHP Debug plugin installed in VSCode
```
Do you think we add a link to the "correct" plugin? I see 2 "PHP Debug" extensions available, adding a link should help figure out which one to install

 ```suggestion
- [PHP Debug](https://github.com/felixfbecker/vscode-php-debug) plugin installed in VSCode
```

brbrr reviewed (changes requested):
Thank you for your contribution! This guide is really helpful. 

I left a few minor comments with some suggestions.

isabellachen commented on docker/README.md:
```diff
@@ -458,3 +458,60 @@ Below are instructions for starting a debug session in PhpStorm that will listen
+
+- PHP Debug plugin installed in VSCode
+- Xdebug Helper for Google Chrome 
+- The easiest Xdebug for Mozilla Firefox.
```
I've changed the link to point to 'Xdebug Helper' for Mozilla Firefox. It's the same as the one for Chrome, except for Firefox.

isabellachen committed:
[not verified] Apply suggestions for vscode debug readme

Co-Authored-By: Yaroslav Kukharuk <i.kukharuk@gmail.com>

isabellachen commented:
@brbrr Thanks for the suggestions, I've committed them and added the link for the Mozilla extension.

isabellachen commented:
@brbrr Hmm, I rebased against the latest master, I probably shouldn't have done that -_- for this task...

isabellachen commented:
@brbrr I think I better make a new branch from master and copy over the text from the readme and make a new PR from there, otherwise there could be trouble...

jeherve committed:
Wrap code snippets in code blocks

jeherve committed:
Update headings and wrap more code

jeherve committed:
Use extension marketplace link for easier installation

jeherve commented:
I took care of the rebase for you (following steps similar to the ones described [here](https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request)), and pushed a few minor changes.

jeherve committed:
Update browser extensions' recommendations

- Include links to extensions for easier installation.
- Update the Firefox add-on as the old one does not appear to be available anymore.

jeherve committed:
Update instructions to add more details

jeherve commented on docker/README.md:
```diff
@@ -458,3 +458,65 @@ Below are instructions for starting a debug session in PhpStorm that will listen
+
+- [PHP Debug](https://marketplace.visualstudio.com/items?itemName=felixfbecker.php-debug) plugin installed in VSCode
+- If you use Google Chrome, install the [Xdebug Helper](https://chrome.google.com/webstore/detail/xdebug-helper/eadndfjplgieldjbigjakmdgkmoaaaoc?hl=en) extension.
+- If you use Firefox, install [Xdebug Helper](https://addons.mozilla.org/en-GB/firefox/addon/xdebug-helper-for-firefox/) add-on.
```
I updated the add-on here as the old one you had previously recommended does not appear to be available in the Firefox add-on library anymore.

jeherve reviewed (approved):
I think this is useful info, it's going to be very helpful for folks using VS Code, thank you. I've made a few minor changes, but overall I think this is good to go.

@brbrr What do you think? Wanna merge this?
